orchestra/google_apps/service.py

```diff
@@ -129,3 +129,47 @@ def delete_folder(self, folder_id):
         except errors.HttpError as error:
             logger.exception('An error occured: %s', error)
             return None
+
+    def list_folder(self, folder_id):
```

Comment:
...and I ran out of time :-). Added it to the journalism workflow cleanup issue.